test/jdk/java/foreign/security.policy

```diff
@@ -0,0 +1,7 @@
+grant {
```

Comment:
To avoid inadvertently granting these permissions to codebases other than the tests, you should change the grant clause to `grant codebase "file:${test.classes}/" {`